lib/filter.g

```diff
@@ -165,7 +165,9 @@ BIND_GLOBAL( "InstallTrueMethodNewFilter", function ( tofilt, from )
         fi;
       fi;
     od;
-    InstallHiddenTrueMethod( tofilt, from );
+    if not GAPInfo.CommandLineOptions.N then
+      InstallHiddenTrueMethod( tofilt, from );
+    fi;
```

Comment:
Wouldn't it be much easier to put the check for the command line option into `InstallHiddenTrueMethod`? Then one can even keep calling `WITH_HIDDEN_IMPS_FLAGS` below, as we'd be guaranteed that there are not hidden implications beyond the regular implications, and almost all changes on the library side could be removed.

Reply:
The `if GAPInfo.CommandLineOptions.N` are introduced such that it is easy to find the places where the behaviour is changed by the `-N` switch.